src/bank.rs

```diff
@@ -41,7 +41,8 @@ use window::WINDOW_SIZE;
 /// but requires clients to update its `last_id` more frequently. Raising the value
 /// lengthens the time a client must wait to be certain a missing transaction will
 /// not be processed by the network.
-pub const MAX_ENTRY_IDS: usize = 1024 * 32;
+pub const NUM_TICKS_PER_SECOND: usize = 10;
+pub const MAX_ENTRY_IDS: usize = NUM_TICKS_PER_SECOND * 32;
```

Comment:
The size of the last_ids queue will be published as part of the solana protocol. Either NUM_TICKS_PER_SECOND should be a power of two, or MAX_ENTRY_IDS should be something that estimates an even chunk of time, (i.e. 10 * 30 = 5 minutes).